compiler/luci-interpreter/src/kernels/Pad.cpp

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Please add Samsung Copyright

Reply:
I stripped the wrong line when copy&pasted 🥇